api/metadata/annotations.go

```diff
@@ -73,3 +73,30 @@ const (
 	// Ideally used in production use cases
 	GitOpsProfile ProfileType = "gitops"
 )
+
+const (
+	DefaultProfile = PreviewProfile
+)
+
+var supportedProfiles = map[ProfileType]ProfileType{DevProfile: DevProfile, PreviewProfile: PreviewProfile, GitOpsProfile: GitOpsProfile}
```

Comment:
Add note that deprecated profile is not supported
```suggestion
// deprecated prod profile is deprecate and not supported, use preview profile
var supportedProfiles = map[ProfileType]ProfileType{DevProfile: DevProfile, PreviewProfile: PreviewProfile, GitOpsProfile: GitOpsProfile}
```
Not sure how exactly we handle deprecated profiles, but it would be good to add there some note. In case we do not need deprecated profile anymore, we can also remove it for next release.